app/assets/javascripts/discourse/app/components/bookmark-local-date.js

```diff
@@ -0,0 +1,57 @@
+import Component from "@ember/component";
```

Comment:
This whole file is the main conflict with my changes here https://github.com/discourse/discourse/pull/11802 that I was talking about on our call @lis2 . Basically I think the only thing I will keep in the bookmark controller is the code to get the post date from the element on the page, otherwise its just a normal tap tile I construct. But it's OK you merge this and I can sort out the conflict on my PR which will be merged later next week.